server/database/fakeData/categoryData.ts

```diff
@@ -0,0 +1,25 @@
+import { Category } from '../models/index';
+
+const allCategories = [
+  {
+    name: 'Food',
+  },
+  {
+    name: 'Money',
+  },
+  {
+    name: 'Clothes',
+  },
```

Comment:
these are not a category, change them so they don't lead to a misunderstanding. and you can just name it `categories` without all.